Snowflake.Data.Tests/SFBaseTest.cs

```diff
@@ -59,7 +59,9 @@ public void SFTestSetup()
             if (testConfigs.TryGetValue(connectionKey, out testConnectionConfig))
             {
                 connectionString = String.Format(connectionStringFmt,
+                    testConnectionConfig.protocol,

```

Comment:
Can you use https as default protocol if not specified? In the example in https://github.com/snowflakedb/snowflake-connector-net/blob/master/README.md, I did not provide protocol and existing customer who is developing against our production environment may break once they update.

Reply:
yes, already set in the constructor of TestConfig